src/query/Querent.js

```diff
@@ -771,6 +798,38 @@ export class Querent {
     return this.q_.all(promises).then(handleCombinedQueryResult_);
   }
 
+  /**
+   * Extend the given bbox with the queryIconPosition values.
+   * @param {!number[]} queryIconPosition The values in px to buffer the bbox.
+   * @param {number} resolution The map view resolution to define the px size correctly.
+   * @param {!import("ol/extent.js").Extent} bbox The bbox to extend.
+   * @return {!import("ol/extent.js").Extent} The extended bbox or null if the queryIconPosition param
+   * is not valid.
+   * @private
+   */
+  extendBboxWithQueryIconPosition_(queryIconPosition, resolution, bbox) {
```

Comment:
Here tolerance is not a min bbox size, but a buffer around queryIconPosition.
The ticket says : A new config will be added queryTouchMin for the min box size for touch devices.
This might create a big bbox with a queryIconPosition + 10px on each side.

Did you reversed the QueryIconPosition somewhere ?
The ticket says : The queryIconPosition specify the positions of the icon => inverse to get the query position.

We do not want the IconPosition bbox, but the bbox to query the real geometry as we now that the use click somewhere in the icon.

=> If the center of the icon center is above the geometry, we need to query below the click. Same for left and right.

Reply:
I've renamed this function to bufferBboxWithQueryIconPosition_ which is more correct